internal/hcl/parser.go

```diff
@@ -12,13 +12,16 @@ import (
 	"github.com/hashicorp/hcl/v2"
 	"github.com/hashicorp/hcl/v2/hclparse"
 	"github.com/hashicorp/hcl/v2/hclsyntax"
-	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 	"github.com/zclconf/go-cty/cty"
 
 	"github.com/infracost/infracost/internal/config"
 	"github.com/infracost/infracost/internal/hcl/modules"
 )
 
+// This sets a global logger for this package, which is a bit of a hack. In the future we should use a context for this.
+var log = logrus.StandardLogger().WithField("parser", "terraform_hcl")
```

Comment:
This is pretty hacky, so I'm not sure about it. The other option would just be to put a `HCL parsing: ` prefix on each log line.

Reply:
think this works for now, perhaps we can actually set it on the `Parser` as a property. However, really not necessary for no

Reply:
I thought about that but it doesn't cover all the logs for this package, so would need to be passed around a fair amount, which I think we'd want to use a context for in the future.